src/overlays/actors/ovl_En_Dns/z_en_dns.h

```diff
@@ -7,26 +7,26 @@
 
 #define EN_DNS_TYPE(thisx) ((thisx)->actor.params)
 typedef enum {
-    /*  0 */ EN_DNS_TYPE_NUTS,
-    /*  1 */ EN_DNS_TYPE_STICKS,
-    /*  2 */ EN_DNS_TYPE_HEART,
-    /*  3 */ EN_DNS_TYPE_SEEDS,
-    /*  4 */ EN_DNS_TYPE_SHIELD,
-    /*  5 */ EN_DNS_TYPE_BOMBS,
-    /*  6 */ EN_DNS_TYPE_ARROWS,
-    /*  7 */ EN_DNS_TYPE_RED_POT,
-    /*  8 */ EN_DNS_TYPE_GREEN_POT,
-    /*  9 */ EN_DNS_TYPE_STICK_UPG,
-    /* 10 */ EN_DNS_TYPE_NUT_UPG,
+    /*  0 */ EN_DNS_TYPE_DEKU_NUTS_5,
+    /*  1 */ EN_DNS_TYPE_DEKU_STICKS_1,
+    /*  2 */ EN_DNS_TYPE_HEART_PIECE,
+    /*  3 */ EN_DNS_TYPE_DEKU_SEEDS_30,
+    /*  4 */ EN_DNS_TYPE_DEKU_SHIELD,
+    /*  5 */ EN_DNS_TYPE_BOMBS_5,
+    /*  6 */ EN_DNS_TYPE_ARROWS_30,
+    /*  7 */ EN_DNS_TYPE_RED_POTION,
+    /*  8 */ EN_DNS_TYPE_GREEN_POTION,
+    /*  9 */ EN_DNS_TYPE_DEKU_STICK_UPGRADE,
+    /* 10 */ EN_DNS_TYPE_DEKU_NUT_UPGRADE,
 } EnDnsType;
 
 typedef enum {
-    /*  0 */ EN_DNS_DIALOG_TYPE_NO_RUPEES,
-    /*  1 */ EN_DNS_DIALOG_TYPE_NO_CAPACITY,
-    /*  2 */ EN_DNS_DIALOG_TYPE_NEW_ITEM,
-    /*  3 */ EN_DNS_DIALOG_TYPE_NO_ITEM,
-    /*  4 */ EN_DNS_DIALOG_TYPE_SUCCESS,
-} EnDnsDialogType;
+    /*  0 */ CANBUY_RESULT_NEED_RUPEES,
+    /*  1 */ CANBUY_RESULT_CAPACITY_FULL,
+    /*  2 */ CANBUY_RESULT_NEW_ITEM,
+    /*  3 */ CANBUY_RESULT_CANT_GET_NOW,
+    /*  4 */ CANBUY_RESULT_SUCCESS,
```

Comment:
```suggestion
    /*  0 */ EN_DNS_CANBUY_RESULT_NEED_RUPEES,
    /*  1 */ EN_DNS_CANBUY_RESULT_CAPACITY_FULL,
    /*  2 */ EN_DNS_CANBUY_RESULT_SUCESS_NEW_ITEM,
    /*  3 */ EN_DNS_CANBUY_RESULT_CANT_GET_NOW,
    /*  4 */ EN_DNS_CANBUY_RESULT_SUCCESS,
```
`EN_DNS_` to avoid misusing the two enums, `EN_DNS_CANBUY_RESULT_SUCESS_NEW_ITEM` just for consistency with `EN_DNS_CANBUY_RESULT_SUCCESS`